docs/book/introduction.md

```diff
@@ -0,0 +1,25 @@
+# Introduction
+
+Giving feedback to a user is an important part of a good application.
+Flash messages are used to notify the user about a successful form submit, failure on saving in the database, wrong authentification credentials or something similar.
```

Comment:
How about this instead: "Flash messages provide notifications to the user, such as for successful form submissions, failure to save data in the database, and incorrect authentication credentials."? I'm not totally sold on it, but feel that it is more succinct.